rpc/client.go

```diff
@@ -436,13 +436,7 @@ func (c *Client) Notify(ctx context.Context, method string, args ...interface{})
 
 // EthSubscribe registers a subscription under the "eth" namespace.
 func (c *Client) EthSubscribe(ctx context.Context, channel interface{}, args ...interface{}) (*ClientSubscription, error) {
-	return c.Subscribe(ctx, "eth", channel, args...)
-}
-
-// ShhSubscribe registers a subscription under the "shh" namespace.
-// Deprecated: use Subscribe(ctx, "shh", ...).
```

Comment:
Since whisper is already deprecated and we don't have `shh` api exposed, I think it's safe to delete it now.